rosdep/base.yaml

```diff
@@ -823,6 +823,14 @@ geographiclib:
     vivid: [libgeographic-dev]
     wily: [libgeographic-dev]
     xenial: [libgeographic-dev]
+geographiclib-tools:
+  debian: [geographiclib-tools]
+  ubuntu:
+    precise: [geographiclib-tools]
```

Comment:
These look to be the same. Is there any reason not to just use `ubuntu: [geographiclib-tools]`?